lang/java/avro/src/main/java/org/apache/avro/generic/GenericData.java

```diff
@@ -94,24 +95,38 @@ public GenericData(ClassLoader classLoader) {
   /** Return the class loader that's used (by subclasses). */
   public ClassLoader getClassLoader() { return classLoader; }
 
-  public Map<String, Conversion<?>> conversions =
+  private Map<String, Conversion<?>> conversions =
```

Comment:
This hasn't been included in a release yet, so it isn't breaking compatibility. I think it was originally an oversight on my part to make them public.